### DISCUSSION TITLE
How can drop move diagonally

### DISCUSSION THREAD

**Discussion by shunnl** [pazusoba/core#35] [Q&A] [answered]
How can drop move diagonally

**Comment by HenryQuan** [comment]
In [`state.cpp`](https://github.com/HenryQuan/pazusoba/blob/master/core/v1/state.cpp), you can uncomment [those line](https://github.com/HenryQuan/pazusoba/blob/9c5830cdb7e80032acd2df4dc1c45ab001ac1d4f/core/v1/state.cpp#L84) to allow diagonal moves but the program will be at least 2x slower.

**Reply by HenryQuan** [reply]
Actually, you just need to comment out `continue` which is this [line](https://github.com/HenryQuan/pazusoba/blob/9c5830cdb7e80032acd2df4dc1c45ab001ac1d4f/core/v1/state.cpp#L87). Now, when it is diagonal moves, the program simply ignores it.

**Comment by shunnl** [comment, marked as answer]
Thank you
